src/main/java/com/google/cloud/datafusion/plugin/successfactors/common/exception/proto/innererror/Application.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.datafusion.plugin.successfactors.common.exception.proto.innererror;
+
+/**
+ * Application class
+ */
+public class Application {
```

Comment:
If we are going to be using it only inside `InnerError` can we keep it inside `InnerError` class?

Reply:
`Application` class moved inside `InnerError` class. 00927432326a05d1fe4a59045fe377ad83711a58